UI/BezelWindow.m

```diff
@@ -183,7 +183,7 @@ -(NSRect) textFrame
 
 -(NSRect) charFrame
 {
-    return NSMakeRect(([self frame].size.width - (3 * lineHeight)) / 2, 7, 4 * lineHeight, 1.2 * lineHeight);
+    return NSMakeRect(([self frame].size.width - (3 * lineHeight)) / 2, 7, 8 * lineHeight, 1.2 * lineHeight);
```

Comment:
It looks like with `4 * lineHeight` (on a tangent about code that was already there before you, using height constants to define width? Ew. 😄) provides 8px leading and trailing margins for `99 of 99`. A `9` is 8px wide (with kerning), and `lineHeight` is `16`. This change adds 64px of width to provide space for at most two 9s, which only need a total 16 px. I haven't played with running the code, but it seems based on this that either the shaded area behind the count is now really wide (looking a bit odd) or that shading is provided by something else and this text field is now a lot wider than the shading area (which opens a door to other problems beyond the 8px margin being eaten to nothing). Is this assessment correct?